package.json

```diff
@@ -4,29 +4,25 @@
   "version": "2.2.0",
   "scripts": {
     "format": "prettier --write .",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "prepare": "husky install"
   },
   "author": "Joel Denning",
   "license": "MIT",
   "homepage": "https://single-spa.js.org/docs/ecosystem-vue.html#vue-cli",
   "repository": "https://github.com/single-spa/vue-cli-plugin-single-spa",
   "bugs": "https://github.com/single-spa/vue-cli-plugin-single-spa/issues",
   "dependencies": {
-    "semver": "^7.3.2",
-    "standalone-single-spa-webpack-plugin": "^1.2.0",
-    "systemjs-webpack-interop": "^2.3.1"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "pretty-quick --staged && eslint ."
-    }
+    "semver": "^7.3.5",
+    "standalone-single-spa-webpack-plugin": "^3.0.0",
```

Comment:
This upgrade is a breaking change to vue-cli-plugin-single-spa.

https://github.com/single-spa/standalone-single-spa-webpack-plugin/releases/tag/v3.0.0

https://github.com/single-spa/standalone-single-spa-webpack-plugin/releases/tag/v2.0.0